src/RootedTrees.jl

```diff
@@ -1413,6 +1415,42 @@ function butcher_representation(t::RootedTree, normalize::Bool = true)
     return result
 end
 
+"""
+    elementary_differential(t::RootedTree)
+
+Returns the elementary differential as a LaTeXString.
+
+"""
+function elementary_differential(t::RootedTree)
+    return latexstring(rec_elementary_differential(t))
+end
+
+"""
+Function used to go recursively through the RootedTree to generate the elementary differential of the tree.
+"""
+function rec_elementary_differential(t::RootedTree)
+    subtree_strings = []
+    for subtree in SubtreeIterator(t)
+        push!(subtree_strings,rec_elementary_differential(subtree))
+    end
+    k = length(subtree_strings)
+    if k == 0 # Special-Case: No Subtree
+        return "f"
+    elseif k == 1 # Special-Case: Just 1 Subtree. No () needed
+        return "f^{\\prime}" * subtree_strings[1]
+    end
+    if k in [2,3] # For first, second and third derivative \prime is used. For the rest just the number of the derivative
```

Comment:
```suggestion
    # For first, second and third derivative \prime is used. 
    # For the rest just the number of the derivative
    if k in (2, 3) 
```

`[2, 3]` will create a new array, requiring a heap allocation. `(2, 3)` is a tuple that can be allocated on the stack and is thus faster.